app/src/main/java/com/asfoundation/wallet/billing/partners/OemIdExtractorV2.kt

```diff
@@ -1,12 +1,15 @@
 package com.asfoundation.wallet.billing.partners
 
 import android.content.Context
-import com.aptoide.injectorextractor.Extractor
+import com.aptoide.apk.injector.extractor.data.Extractor
+import com.aptoide.apk.injector.extractor.data.ExtractorV1
+import com.aptoide.apk.injector.extractor.data.ExtractorV2
 import io.reactivex.Single
 import java.io.File
 
 class OemIdExtractorV2 @JvmOverloads constructor(private val context: Context,
```

Comment:
Why are we using @JvmOverloads here?

Reply:
Because otherwise default parameter values are not available in Java.
Yep.. that ancient first-class citizen.. :)